Objects/StrategyCardAutomator.ttslua

```diff
@@ -879,15 +780,13 @@ function _closeMenu(color, card)
 		UI.setAttribute(card, "active", "false")
 		if not popup then
 			broadcastToAll("All players have responded", "Black")
-			if _useWorkQueue then
-				_clearWorkQueue(card)
-			end
```

Comment:
Remove old code for _workQueue (that was meant to deal action and secret cards in the correct order).
This code was not being used.